File: foundation/foundation 3 month/lesson3/classwork/ex_5.py
k=input("Paloni kiriting: ")
f=len(k)
katta=0
kichik=0
raqam=0
belgi=0
a=0
while a<f:
	if f<8:
		break
	elif(f>=8):
		if k[a]>='A' and k[a]<='Z':
			katta+=1	
		elif k[a]>='0' and k[a]<='9':
			raqam+=1
		elif k[a] in "@!#%&$":
			belgi+=1
		elif k[a]>='a' and k[a]<='z':
			kichik+=1
	a+=1
if katta>0 and kichik>0 and raqam>0 and belgi>0:
	print("Juda kuchli parol")
elif katta==0:
	print("Katta harf yetamadi")
elif kichik==0:
	print("Kichik harf yetmadi:")
elif raqam==0:
	print("Raqam yetamydi")
elif belgi==0:
	print("BElgi yetmaydi")

File: full stack/full stack 1 month/lesson6/homework/ex_5.js
let ans = "18fF."
let raqam = 0
let belgi = 0
let harf = 0
for (let i = 0; i<ans.length;i++)
{
    if(ans[i]>=0 && ans[i]<=9)
    {
        raqam+=1
    }
    else if(ans[i]>="A" && ans[i]<="Z" || ans[i]>="a" && ans[i]<="z")
    {
       harf+=1
    }
    else
    {
        belgi+=1
    }
}
console.log(`Raqam: ${raqam}, Belgi: ${belgi}, Harf: ${harf}`);